runtime/include/chplsys.h

```diff
@@ -36,4 +36,9 @@ int chpl_getNumLogicalCpus(chpl_bool accessible_only);
 //
 c_string chpl_nodeName(void);
 
+//
+// Returns the value of a CHPL_RT_* environment variable, with default.
+//
+chpl_bool chpl_env_getBool(const char*, chpl_bool);
```

Comment:
I think this needs a slightly different name; something to indicate it's get a CHPL_RT_* variable and not CHPL_*. 

Maybe chpl_get_rt_env_bool (or chpl_rt_get_env_bool, but I don't think that's as obvious)? This would kind of mimic what qthreads did with qt_internal_get_env_bool() and siblings